Codeinwp/neve
# AMP Integration

### Summary
Added AMP support.


### Will affect visual aspect of the product

YES

### Test instructions


- Install the latest AMP RC from https://github.com/Automattic/amp-wp/releases
- Make sure it validates
- Test if interactions load ok on AMP.


Closes #35.
